### ISSUE TITLE
Editing Multiple Frames at the Same Time

### ISSUE THREAD

**Issue by Luxapodular** [Wicklets/wick-editor#162]
**Is your feature request related to a problem? Please describe.**
In other animation programs, it is possible to select multiple frames and edit them at the same time. Often times, it is tied to onion skinning as well, allowing you to see changes on multiple frames at once.

**Describe the solution you'd like**
A user can select multiple frames, and when the draw onto the canvas that change will occur on all frames.

**Comment by Capploo** [comment]
Yes, my problem is that i started an animation without noticing that I'm using a wrong canvas size. As i changed it's size to 800x450px what i mainly use, the animation moved upwards. That's why i wanted to drag every frame down together - easier and more precise than doing it frame by frame.
The solution I'd like to have is an opportunity to select multiple frames and resize/drag and drop them together.
